easybuild/toolchains/compiler/inteliccifort.py

```diff
@@ -71,13 +71,11 @@ class IntelIccIfort(Compiler):
 
     # used when 'optarch' toolchain option is enabled (and --optarch is not specified)
     COMPILER_OPTIMAL_ARCHITECTURE_OPTION = {
-        systemtools.INTEL : 'xHost',
-        systemtools.AMD : 'xHost',
+        systemtools.X86_64: 'xHost',
```

Comment:
@geimer hmm, I'm not so sure we want to drop the distinction between `AMD` and `INTEL` here... right now, it doesn't matter, but it may sometime in the future?